<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>svg</class>
 <widget class="QWidget" name="svg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>svg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushDebutTracer">
     <property name="text">
      <string>Debut tracer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushDebGene">
     <property name="text">
      <string>Generation SVG</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushContinueTracer">
     <property name="text">
      <string>Continuer Tracer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushFinGene">
     <property name="text">
      <string>Fin Generation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
